setup.py

```diff
@@ -11,12 +11,12 @@
     license='MIT',
     description='Very flexible date parsing and normalization utilities',
     long_description=__long_description__,
-    author='Open Knowledge Foundation',
-    author_email='info@okfn.org',
-    url='http://github.com/okfn/flexidate/',
```

Comment:
Yes. That was part of my other repository. I forgot to change it, hence I made the other commit. Sorry for the confusion